### ISSUE TITLE
Please update your demo

### ISSUE THREAD

**Issue by jdriesen** [leonardovilarinho/vue-acl#6]
Ho Leonardo,

Can I kindly ask to update (the source-code) of your demo a bit, please ?

I'll try to be a bit more clear...
Normally, when you use 'Vue-router', you have a folder  **\src\router** with **index.js** in it...
This is NOT the case you your example...

This is a bit confusing for newbies like me...
So, can I kindly ask to update your demo ?

Thanks in advance,

Johnny

PS.: As  you can see, I became a fan of your modules :) :)

**Comment by jdriesen** [comment]
Hi Leonardo,

Please keep this issue a bit ON HOLD.
I will make a very simple sample application today, so you can perfectly understand my problem.

Please give me a couple of hours.

Meanwhile, kind regards,
Johnny

**Comment by jdriesen** [comment]
Hi Leonardo,

I created a small demo project.
It is SUPER BASIC.
I tried to do the best to make it as clear as possible.
[It can be seen here](http://test05.ireporter.be)

What's the idea of this project...
On initial startup, you're NOT logged in (the NotLoggedIn component is shown)
I can login as ADMINISTRATOR, or as USER
The ADMINISTRATOR has ALL access
The USER has LIMITED access.

The Sources
[The Sources](https://github.com/jdriesen/VueJS-2-ACL-Test)

**My Problem**:
I don't know how to make you component working..
(I don't know where (in which files) to implement the IMPORT and the Vue.use commands...

**My Question**:
Can I kindly ask you to give me a hand ?

**My offer**
Feel free to use this little project as a demo for your software.
I think it will clarify a lot for other developers.

Thanks in advance for your time and effort !!
Kind regards,
Johnny Driesen
Belgium
